test/data/test_builtin_datasets.py

```diff
@@ -38,20 +38,6 @@ def test_raw_ag_news(self):
         self._helper_test_func(len(test_iter), 7600, next(test_iter)[1][:25], 'Fears for T N pension aft')
         del train_iter, test_iter
 
-    @parameterized.expand(
```

Comment:
A n00b question. Why did we remove this and how this change relate to migration of AmazonReviewPolarity to datapipes ?